src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -884,116 +921,125 @@ int wm_vuldet_send_agent_report(sqlite3 *db, OSHash *cve_table, agent_software *
             // As we consider the NVD as our source of trust, within this statement we will
             // add information only if that information was not already added by the NVD
             // report, or if the information is not repeated.
-            if (pkg->feed & VU_SRC_OVAL) {
```

Comment:
This condition was avoiding the possibility to add extra information to the alerts. Now the behavior is, if the alert was generated by the NVD, we fill the information and then we add here any missing field.
If the alert was not generated by the NVD, we directly add here all the information from the vendor feeds.